test/units/modules/crypto/test_luks_device.py

```diff
@@ -85,52 +97,62 @@ def run_command_check(self, command):
+    ("dummy", "dummy", "name", "name", "present", None, False),
+    ("dummy", "dummy", "name", "name", "absent", None, False),
+    ("dummy", "dummy", "name", "name", "opened", None, False),
+    ("dummy", "dummy", "name", "name", "closed", None, True),
+    (None, "dummy", "name", "name", "closed", None, True),
+    ("dummy", "dummy", None, "name", "closed", None, True),
+    (None, "dummy", None, "name", "closed", None, False))
+
+# device, key, new_key, state, label, expected
 LUKS_ADD_KEY_DATA = (
-    ("dummy", "key", "new_key", "present", True),
-    (None, "key", "new_key", "present", False),
-    ("dummy", None, "new_key", "present", False),
-    ("dummy", "key", None, "present", False),
-    ("dummy", "key", "new_key", "absent", "exception"))
-
-# device, remove_key, state, expected
+    ("dummy", "key", "new_key", "present", None, True),
+    (None, "key", "new_key", "present", "labelName", True),
+    (None, "key", "new_key", "present", None, False),
+    ("dummy", None, "new_key", "present", None, False),
+    ("dummy", "key", None, "present", None, False),
+    ("dummy", "key", "new_key", "absent", None, "exception"))
+
+# device, remove_key, state, label, expected
 LUKS_REMOVE_KEY_DATA = (
-    ("dummy", "key", "present", True),
-    (None, "key", "present", False),
-    ("dummy", None, "present", False),
-    ("dummy", "key", "absent", "exception"))
+    ("dummy", "key", "present", None, True),
+    (None, "key", "present", None, False),
+    (None, "key", "present", "labelName", True),
+    ("dummy", None, "present", None, False),
+    ("dummy", "key", "absent", None, "exception"))
 
 
-@pytest.mark.parametrize("device, keyfile, state, is_luks, expected",
-                         ((d[0], d[1], d[2], d[3], d[4])
+@pytest.mark.parametrize("device, keyfile, state, is_luks, label, expected",
+                         ((d[0], d[1], d[2], d[3], d[4], d[5])
                           for d in LUKS_CREATE_DATA))
-def test_luks_create(device, keyfile, state, is_luks, expected, monkeypatch):
+def test_luks_create(device, keyfile, state, is_luks, label, expected,
+                     monkeypatch):
     module = DummyModule()
 
     module.params["device"] = device
     module.params["keyfile"] = keyfile
     module.params["state"] = state
+    module.params["label"] = label
 
     monkeypatch.setattr(luks_device.CryptHandler, "is_luks",
                         lambda x, y: is_luks)
     crypt = luks_device.CryptHandler(module)
-    conditions = luks_device.ConditionsHandler(module, crypt)
-    assert conditions.luks_create() == expected
+    if device is None:
+        monkeypatch.setattr(luks_device.Handler, "get_device_by_label",
+                            lambda x, y: [0, "/dev/dummy", ""])
+    try:
+        conditions = luks_device.ConditionsHandler(module, crypt)
+        assert conditions.luks_create() == expected
+    except ValueError:
+        assert expected == "exception" or expected is False
```

Comment:
```suggestion
        assert expected == "exception"
```
If `expected` is `False`, an exception should not be accepted.